modules/commerce_option_weight/commerce_option_weight.module

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * @file
+ * Contains hooks related to commerce option weight module.
+ */
+
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\Core\Routing\RouteMatchInterface;
+
+
+/**
+ * Implements hook_entity_base_field_info().
+ */
+function commerce_option_weight_entity_base_field_info(
+  EntityTypeInterface $entity_type
+) {
+  $fields = [];
+
+  if ($entity_type->id() != 'commerce_option_option_value') {
+    return $fields;
```

Comment:
Let's just return nothing `return;` because I worry that return an empty array might cause issues with other modules that extend this hook.